docs/sources/style-guide/write-for-developers/index.md

```diff
@@ -0,0 +1,153 @@
+Pay particular attention to properly formatting the elements of code.
+
+### Auto-generated documentation
+
+The advantages of automating documentation programmatically are well-known, and include increased consistency and a reduction in human-made errors.
+But behind every line of auto-generated content is a human author who is responsible for following the [style guide]({{< relref "../../style-guide/" >}}).
+
+When writing documentation that will be used by an auto-generated program to create publishable content, keep the following things in mind:
+
+An auto-generation tool can parse syntax, but when writing documentation that will be used to create automated content, it is up to you to add actionable insights. 
+- What are the caveats, edge cases, and side effects?
+- What is the bigger picture that is not self-evident in the code? 
+- In short, what is everything that a developer needs to know to use the code?
+
+> **Note:** It might be difficult to integrate auto-generated content with other documentation, such as relevant sections of _Get started_ guides, tutorials, or detailed code examples. You might need to ask a Docs team member to ensure that your content is properly cross-referenced.
+
+### Elements of API references
+
+Properly document the most common elements of an API reference, such as the title, parameters, return values, and so on. The following suggestions will help you to write more complete and consistent documentation:
+
+| Element                                               | Description                                                           |
+| ------------------------------------------------- | ----------------------------------------------------------------------- |
+| Title and description       | The name of the element and a description of it in one or two sentences. Use back ticks (``) for API names, classes, methods, and so forth, so they display in a fixed-width font. |
+| Syntax | The code signature that defines the element. If it is possible to use multiple programming languages, provide the syntax for each. Put the signature in `code font`. |
+| Parameters | If the element has parameters, specify their descriptions, data types, and state whether they are optional or required. Put the parameters in _italic_. |
+| Return values| If the element returns a value, describe the range of possibilities and the data type. |
+| Error codes| Describe errors or exceptions and the conditions under which they occur. If possible, provide a way to resolve the issue. |
+| Comments | Describe any important information that hasn't been previously included in the title, description, syntax, parameters, or return values. For example, you might explain non-obvious context, make a comparison to similar elements, or provide cautionary notes about potential gotchas. |
+
+Additional tips:
+- Write concisely. Don't say "The `AddUser` method adds a user." or "This method adds a user." when "Adds a user." will do.
```

Comment:
i appreciate and agree with the general point here, but the particular examples chosen will actually run afoul of Go linting standards - if there's a method called `AddUser`, it is expected that the corresponding docs begin with "AddUser", allowing "A" or "The" before it.

This seems likely to create an unnecessary conflict in a Go developer's mind. But the problem isn't the principle, just the particular example - if you're able to reword that first example to illustrate the concept without incidentally bumping into this Go convention, it would sidestep the whole problem.

Reply:
I have replaced with:

- Remember to write concisely. Don't say "This method adds a user." when "Adds a user." will do. However, you may say, "The AddUser method adds a user." if needed to avoid a Go linting error.